### DISCUSSION TITLE
How do you switch to normal mode in the search window?

### DISCUSSION THREAD

**Discussion by niamleeson** [ibhagwan/fzf-lua#1144] [General]
In telescope, you can switch between normal and insert modes. Is it possible to do it with this plugin as well?

**Comment by ibhagwan** [comment]
Fzf is a terminal window, in neovim, to exit terminal (insert) mode you need to press `<C-\><C-n>`.

**Reply by gengdz** [reply]
@ibhagwan i can use `<C-\><C-n>` enter normal mode, but I can't use delete-related commands like `x` 

For example: I wrote ibbhagwan/fzf-lua but added an extra 'b'. I want to delete it. What should I do?

**Reply by ibhagwan** [reply]
As I mentioned, fzf is a terminal window and as such you can’t delete characters with `x`, IMHO normal mode is pretty useless with terminal buffers, if you wanted you could create elaborate mappings that send keys and somewhat achieve what you’re after but I wouldn’t recommend it.

**Reply by ibhagwan** [reply]
That said, fzf has useful binds for line editing, for example `ctrl-w` to delete last word, `ctrl-u` to clear the line, etc, see `man Fzf` for the full list.
